effektio_flutter_sdk/lib/effektio_flutter_sdk_ffi.dart

```diff
@@ -655,52 +657,6 @@ class Api {
     return tmp9;
   }
 
-  String echo(
```

Comment:
this seems unrelated, but not a problem.

Reply:
It must be outdated. Should I add generated files to gitignore?